loader/db.go

```diff
@@ -95,10 +95,12 @@ func (conn *DBConn) querySQL(ctx *tcontext.Context, query string, args ...interf
 			return ret, err
 		})
 	if err != nil {
-		ctx.L().Error("query statement failed after retry",
-			zap.String("query", utils.TruncateString(query, -1)),
-			zap.String("argument", utils.TruncateInterface(args, -1)),
-			log.ShortError(err))
+		if !utils.IsContextCanceledError(err) {
```

Comment:
1. should we need to filter other logs in this file? like
```go
err = conn.resetConn(ctx)
if err != nil {
    ctx.L().Error("reset connection failed", zap.Int("retry", retryTime),
        zap.String("query", utils.TruncateInterface(query, -1)),
        zap.String("arguments", utils.TruncateInterface(args, -1)),
        log.ShortError(err))
    return false
}
```

can this `err` also become `ContextCanceled`.

2. how about adding a method in `pkg/log/Logger` to do this check (err != ***)?